source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -0,0 +1,68 @@
+.. include:: ../../variables.rst
+.. _release-notes:
+
+=============
+Release notes
```

Comment:
Looks good with this new layout! I guess there is a plan on how to release this new layout, so we don't end up with missing notes or empty drop downs?

Reply:
I think I would enter a note about there not being any recent release notes and to look at the old notes, or hide the drop down instead of it being empty in the transitioning phase. It kind of looks like it's not working if it's empty.